src/main/kotlin/racingcar/controller/RacingController.kt

```diff
@@ -0,0 +1,52 @@
+package racingcar.controller
+
+import racingcar.model.*
+import racingcar.view.InputView
+import racingcar.view.OutputView
+
+class RacingController {
+    fun race() {
+        val carList = getCarList()
+        val attemptCount = getAttemptCount()
+        showResult()
+        processRacing(carList, attemptCount)
+        showWinners(carList)
+    }
+
+    private fun getCarList(): CarList {
+        OutputView.printStartMessage()
+        val input = InputView.getCars()
+        return CarList(input)
+    }
+
+    private fun getAttemptCount(): AttemptCount {
+        OutputView.printAttemptMessage()
+        val input = InputView.getAttempt()
+        return AttemptCount(input)
+    }
+
+    private fun showResult() {
+        OutputView.printResultMessage()
+    }
+
+    private fun processRacing(carList: CarList, attemptCount: AttemptCount) {
+        val round = Round(RandomNumberGenerator())
+        repeat(attemptCount.count) {
+            round.play(carList.cars)
+            showRoundStatus(carList)
+        }
+    }
+
+    private fun showRoundStatus(carList: CarList) {
+        val racingStatus = RacingStatus()
+        val carDtoList = racingStatus.getCurrentRacingStatus(carList.cars)
+        carDtoList.forEach { OutputView.printPosition(it.name, it.position) }
```

Comment:
인협님 방식도 코드가 간략해진다는 장점이 있지만!!! it말고 car를 사용하면 가독성 측면에서는 이 방법이 조금 더 좋지 않을까? 생각에 남겨봅니다! 이건 아마 취향차이..? 일거 같기도 합니다!! 
```suggestion
        carDtoList.forEach { car -> 
            OutputView.printPosition(car.name, car.position)
        }
```

Reply:
이런 사소한 부분까지 챙길 수도 있겠네요. 감사합니다